CannibalisticZombies/Assets/PlayerCharacterController.cs

```diff
@@ -0,0 +1,45 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerCharacterController : MonoBehaviour
+{
+    public float moveSpeed;
```

Comment:
We do not need to keep the variables `moveSpeed` and `orientation` public since we wont be accessing it from other scripts. Instead, we can make it private and add the attribute `[SerializeField]` at the front in order to expose it into the inspector.

Ex: `[SerializeField] private float moveSpeed;`